crates/re_viewer/src/misc/caches/mod.rs

```diff
@@ -92,6 +89,8 @@ impl TensorStats {
         declare_tensor_range_float!(tensor_range_f32, f32);
         declare_tensor_range_float!(tensor_range_f64, f64);
 
+        // TODO(854) re-enable once we have F16 tensors
+        /*
         #[allow(clippy::needless_pass_by_value)]
```

Comment:
instead of commenting out code, disabled it. Out-commented code will rot!

```suggestion
        #[allow(unused_code)] // TODO(854) re-enable once we have F16 tensors
        #[allow(clippy::needless_pass_by_value)]
```